src/components/ProjectDetail/TaskListBoard/index.tsx

```diff
@@ -20,27 +20,18 @@ export const categories = [
 
 function TaskListBoard({ tasks }: TaskListBoardProp) {
    const [tasksMap, setTasksMap] = useState<TasksMap>({});
+
+   // add sequence of tasks
+   function addSequence(task: Task[]) {
+      task.forEach((task, index) => (task.sequence = index));
+   }
```

Comment:
cái này a sót, e sử dụng useCallback  nhé, để tránh n luôn render khi props, state thay đổi

Reply:
Cái function này em để thêm cái **sequence** cho task trong từng cột ý anh, thì em gọi nó sau khi mình kéo thả xong sẽ update cái sequence của task trong cột theo cái **status** mình kéo vào, xong rồi em mới **setTaskMap**, thì em xem lại em thấy nó không có phụ thuộc vào props hay states

Reply:
- Cái này k quan trọng lắm ở đây (e có thể bỏ qua k cần sửa) vì component mình khá nhẹ, chỉ để e hiểu rõ hơn về useCallback thôi. Khi e dùng useCallback thì nó sẽ không bị tính lại hàm khi render lại, khi k dùng nó sẽ tính lại hàm này. Với component ít hàm thì việc tính toán này cũng k ảnh hưởng lắm. E có thể xem và thử với ví dụ trong đây để hiểu hơn về useCallback.

Reply:
à quan trọng hơn là e sửa lại tên biến nhé: để tasks số nhiều (vì nó là mảng :smiley: )

Reply:
Vâng ạ , em sửa rồi ạ